features/FooterUpdate.feature

```diff
@@ -0,0 +1,66 @@
+@owner:derek
+@author:lilla
+Feature: Docs Footer Update
+
+    New responsive design and links for the footer on the docs pages.
+
+    Background: A user arrives on the SaaSquatch help center page
+        Given the user stays on the 'https://docs.saasquatch.com/' pages
```

Comment:
Not fully sure what you mean by `the user stays`, something like `Given a user viewing a 'https://docs.saasquatch.com/' page`

Reply:
I meant everything under that URL if they stay within the docs site. But I removed the whole background.